tests/ru/climate_HassClimateGetTemperature.yaml

```diff
@@ -23,3 +23,47 @@ tests:
           - Кухня
           - Кухне
           - Кухонный
+
+  - sentences:
+      - Сколько градусов на термостате в зале?
+    intent:
+      name: HassClimateGetTemperature
+      slots:
+        name:
+          - Термостат
+        area:
+          - Зал
+          - Зале
+    response:
+      - 24 градуса
+
+  - sentences:
+      - Какая температура термостата?
+      - Какая температура на термостат 1?
+      - Какая температура на термостат 2?
+      - Какая температура на термостат 5?
+      - Какая температура на термостат 11?
+      - Какая температура на термостат 21?
+      - Какая температура на термостат 22?
+      - Какая температура на термостат 25?
+    intent:
+      name: HassClimateGetTemperature
+      slots:
+        name:
+          - Термостат
+          - Термостат 1
+          - Термостат 2
+          - Термостат 5
+          - Термостат 11
+          - Термостат 21
+          - Термостат 22
+          - Термостат 25
+    response:
+      - 24 градуса
+      - 1 градус
+      - 2 градуса
+      - 5 градусов
+      - 11 градусов
+      - 21 градус
+      - 22 градуса
+      - 25 градусов
```

Comment:
I'm not seeing point of having an eight exact the same tests. I get if it will be two, but eight?

Reply:
Oh, now I get it, you're testing different kind of values. No questions here anymore.